independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartProjectGeneration.java

```diff
@@ -47,11 +48,19 @@ static void generateProject(final CodestartProjectDefinition projectDefinition,
             processor.process(codestart);
         }
         processor.writeFiles();
+
         log.info("\napplying codestarts...");
-        log.info(projectDefinition.getCodestarts().stream()
-                .map(c -> c.getType().getIcon() + " "
-                        + c.getName())
-                .collect(Collectors.joining("\n")));
+        if (!OS.WINDOWS.isCurrent()) {
```

Comment:
The code is easier to read if you invert this condition

Reply:
Well it's maybe easier to ready if the condition is in the `getIcon()` no?

Reply:
Sure, that would make the `getAlternativeIcon` no longer necessary

Reply:
done